src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import './App.css'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import { createStore, applyMiddleware } from '@reduxjs/toolkit'
+import { Provider } from 'react-redux'
+// import promiseMiddleware from 'redux-promise'
+import CitySearch from './components/CitySearch'
```

Comment:
CitySearch and Forecast had incorrect paths, should be
import CitySearch from './components/citySearch'
import Forecasts from './components/forecasts'